ambry-protocol/src/test/java/com.github.ambry.protocol/RequestResponseTest.java

```diff
@@ -165,33 +241,28 @@ public void putRequestResponseTest() throws IOException {
         new BlobProperties(blobSize, "serviceID", "memberId", "contentType", false, Utils.Infinite_Time,
             Utils.getRandomShort(TestUtils.RANDOM), Utils.getRandomShort(TestUtils.RANDOM));
     testPutRequest(clusterMap, correlationId, clientId, blobId, blobProperties, userMetadata, BlobType.DataBlob, blob,
```

Comment:
minor: Might be better off to do invalid version test as one of the components in this function than in `testPutRequest`

Reply:
Doesn't make a lot of difference I think and I just found this convenient.